src/com/chocohead/eumj/GuideThings.java

```diff
@@ -47,5 +50,14 @@ public PagePosition handleMouseClick(int x, int y, int width, int height, PagePo
 				}
 			});
 		});
+		Object[] engines = {
+            EngineMod.engine.getItemStack(Engine_TEs.slow_electric_engine),
+            EngineMod.engine.getItemStack(Engine_TEs.regular_electric_engine),
+            EngineMod.engine.getItemStack(Engine_TEs.fast_electric_engine),
+            EngineMod.engine.getItemStack(Engine_TEs.quick_electric_engine),
+            EngineMod.engine.getItemStack(Engine_TEs.adjustable_electric_engine),
+		};
+        GuideGroupManager.addEntries("buildcraft", "pipe_power_providers", engines);
+        GuideGroupManager.addEntries("buildcraft", "full_power_providers", engines);
```

Comment:
Unfortunately my editor uses spaces and not tabs, would you be alright cleaning this up?

Reply:
Yeah that's be fine